templates/react-js/src/App.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export function App() {
+  return (
+    <React.StrictMode>
```

Comment:
I think it's better to have `React.StrictMode` wrap the App component in index.js (reduces clutter in App)